cmake/schemes/url_sha1_boost_library.cmake.in

```diff
@@ -102,6 +102,23 @@ else()
   hunter_fatal_error("TODO: set toolset for boost" WIKI "error.boost.toolset")
 endif()
 
+if("@ANDROID@")
+  set(toolset_version ndk)
+endif()
+
+get_directory_property(defs COMPILE_DEFINITIONS)
+foreach(def ${defs})
+    set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -D${def}")
```

Comment:
Ok, maybe adding the `DEFINITION_OPTION` would be pushing it too far.
I will put the `if("@ANDROID@")` guards around the `get_directory_property(...)` to keep it simpler